src/index.ts

```diff
@@ -74,33 +72,18 @@ import { resolveV3Document } from './v3/parser';
  *
  */
 export default async function bundle(files: string[], options: any = {}) {
-  if (typeof options.base !== 'undefined') {
-    options.base = toJS(options.base);
-    await parse(options.base);
-  }
-
```

Comment:
Why are removing these options?

Reply:
`baseDir` and `resolveBaseFileDir()` were related to resolving `$ref`s in `$.channels.*.messages.*` and `$.operations.*.messages.*` only: full dereferencing functionality was rewritten and is now handled by `@apidevtools/json-schema-ref-parser` and I didn't catch usecases when specifying an absolute path was needed. Have I missed something?

Reply:
Is this related to
https://github.com/asyncapi/bundler/issues/136
?

Reply:
Yeah, I see in this PR, you have removed the changes from #136, which is all alright if you are improving the solution. 


When the spec files were in a separate directory, bundler was having problem resolving the paths. So `baseDir` was to solve that issue to let bundler know where to look for the files.

Reply:
I returned and improved the `baseDir` functionality, but now it is required to pass to the `Bundler` paths of AsyncAPI Documents as an array of strings. Bundler changes the working directory to `baseDir` **and only then** starts reading the AsyncAPI Documents from the filesystem.
Should this change be mentioned somewhere, taking into account that `Bundler` is only v0.5.0?

Reply:
Added explanation of how `baseDir` currently works to [README.md](https://github.com/asyncapi/bundler/pull/147/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R218).